src/win/tty.c

```diff
@@ -2255,25 +2250,67 @@ int uv_tty_reset_mode(void) {
   return 0;
 }
 
+static int uv__set_vtp(HANDLE handle, BOOL vtp) {
+  DWORD dwMode = 0;
+
+  if (!GetConsoleMode(handle, &dwMode)) {
```

Comment:
Can you explicitly check `0 == GetConsoleMode(handle, &dwMode)`. Same thing applies to the `SetConsoleMode()` call below and other checks throughout the PR.

Reply:
I have no right commenting, since I'm not a maintainer, but a lot of the other Console calls in this file .e.g `GetConsoleScreenBufferInfo aren't using this convention, thought I'd mention, maybe this is not important